CONTRIBUTING.md

```diff
@@ -80,35 +78,30 @@ The External Adapter will do some processing, often request data from an API, an
-3. Using the generated fixture data from step 2, write a `fixtures.ts` file to return the mock data instead.
-4. Run the tests again with nock recording disabled (`unset RECORD`). API requests should now be intercepted and mocked using the fixture data. Be sure to run tests with the `--updateSnapshot` flags to update the integration snapshot if necessary.
+1. Setup the test, see the [coingecko](./packages/sources/coingecko/test/integration/adapter.test.ts) test for a code sample.
+2. Make a request to the Data Provider and copy the response. For HTTP requests, you can use tools like `curl`. Change/mock any sensitive data that it contains.
+3. Using the mock data from step 2, write a `fixtures.ts` file to return the mock data instead.
+4. Run the tests. API requests should now be intercepted and mocked using the fixture data.
 
 ### Testing WebSocket Messages
 
-1. Run your tests, using live API endpoints with recording on (`export RECORD=true`) and with a TTL for the connections long enough for the normal requests to go through, but lower than the jest timeout (this will depend on the test, but one example would be `export WS_SUBSCRIPTION_TTL=3000`)
-2. You will see a log statement with the message "Recorded WebSocketMessages: {JSON Object}". This JSON object contains all the WebSocket messages sent and received, but they are not printed as code as in the case of the Nock features, due to their asynchronous nature.
-3. Using the recorded messages, write a `fixtures.ts` file with "Exchanges", i.e. request and response(s) pairs that will be asserted as part of your test (see this [ncfx test fixtures example](./packages/sources/ncfx/test/integration/fixtures.ts)).
-4. Write your tests (example in [ncfx adapter test](./packages/sources/ncfx/test/integration/adapter.test.ts)) using the helper functions from the [test-helpers](./packages/core/test-helpers/src/websocket.ts) package. Note the necessary setup performed in the `beforeAll` function. Also note the path for the `WebSocketClassProvider` import, it has to be that one due to the way Singleton patterns work (or rather don't work) across dependencies.
-5. Finally, run your tests with recording disabled (`unset RECORD`). The WebSocket connection should be replaced and mocked.
+1. Setup the test, see the [ncfx](./packages/sources/ncfx/test/integration/adapter.test.ts) test for a code sample.
+2. Make a request to the Data Provider and copy the websocket messages. For websocket requests, you can use tools like [wscat](https://github.com/websockets/wscat). Change/mock any sensitive data that it contains.
+3. Using the copied messages, write a `fixtures.ts` file. Create mock function in `fixtures.ts` that will return mock messages. See [ncfx test fixtures example](./packages/sources/ncfx/test/integration/fixtures.ts).
+4. Run the tests. Websocket requests should now be intercepted and mocked using the fixture data.
```

Comment:
We may also want to include a step here to test and mock the auth message as well. Probably between step 1 and 2